src/main.ts

```diff
@@ -65,13 +66,21 @@ async function run(): Promise<void> {
             )
         }
 
-        info(`Sorting packages according to dependencies`)
-        const sorted_packages = sortPackages(packages)
+        let sorted_packages
+        if (!no_verify) {
+            info(`Sorting packages according to dependencies`)
+            sorted_packages = sortPackages(packages)
+        } else {
+            sorted_packages = Object.keys(packages)
```

Comment:
I think that complete bypassing sorting isn't a best solution. It may cause problems with publishing (for ex. in case of `rquickjs`). In my opinion skipping `dev-dependencies` when sorting is much better.

Reply:
If verification is disabled theoretically the order is irrelevant since the deps are only downloaded and checked during cargo publish packaging (which is skipped when --no-verify is used).

Reply:
Course, but in that case `no-verify` will be required to publish `rquickjs`.

Actually `--no-verify` arg is used here to speed-up publishing by skipping complex verification because it already executed in previous steps of CI workflow.